eni.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_network_interface" "default" {
```

Comment:
This `aws_network_interface` is used for `additional` `aws_eip`.
Let's rename it to `additional`.